Pull latest changes from opendata-stuttgart

DavidLemayian opened the pull request:
Pull latest changes from opendata-stuttgart/feinstaub-map.

ricki-z committed:
leaflet-hash added

some more cities and countries added

ricki-z committed:
Portugal added

ricki-z committed:
Nigeria added

ricki-z committed:
new countries

Peru, Kenya

ricki-z committed:
new countries and cities added

ricki-z committed:
filter ectremely high values

filter sensors with PM10 >= 1999 and PM2.5 >= 999
China added

ricki-z committed:
Sweden, Slovenia added

ricki-z committed:
graphs changed

switching between cells with man open graphs created massive amount of open connections

ricki-z committed:
added south korea, tanzania

ricki-z committed:
added new sensors

Kyrgyzstan added
switched to one map server

ricki-z committed:
new countries added

Bilivia, Brazil, Egypt, Latvia, Norway

ricki-z committed:
new countries added

adding support for relative path to map

ricki-z committed:
changed back to own tiles proxy

no access to files outside domain luftdaten.info
added new places

ricki-z committed:
wrong IDs in hexagons after moving and zooming resolved

ricki-z committed:
new countries added

arouanet committed:
Add Rennes, France to the list of places

ricki-z committed:
new places

ricki-z committed:
Merge pull request #59 from arouanet/rennes-places

Add Rennes to the list of places

DavidLemayian commented on index.html:
```diff
@@ -4,10 +4,11 @@
+    <link rel="stylesheet" href="https://maps.luftdaten.info/fonts/css?family=Roboto:400,100,300,500,700,900,400italic,100italic,300italic,700italic,500italic,900italic" />
+    <link href='https://maps.luftdaten.info/fonts/css?family=Roboto+Mono:700,400,100,300' rel='stylesheet' type='text/css'>
+    <link href="https://maps.luftdaten.info/fonts/css?family=Roboto+Slab:400,700" rel="stylesheet">
+    <link href="https://maps.luftdaten.info/fonts/icon?family=Material+Icons" rel="stylesheet">
```
@chegejames Any idea why/how they're doing this proxy even though hosted on Github Pages?

ricki-z replied:
@DavidLemayian We do this to limit the user access to other servers. So the user access won't be visible to Google or others (we have disabled logging on our server).
It's implemented as a apache proxy service.

DavidLemayian replied:
Oh.. Interesting. Thanks.

DavidLemayian commented on src/map.vue:
```diff
@@ -16,7 +16,8 @@ export default {
 			})
-			leaflet.tileLayer('https://{s}.tiles.madavi.de/{z}/{x}/{y}.png', {
+//			leaflet.tileLayer('https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', {
+			leaflet.tileLayer('https://maps.luftdaten.info/tiles/{z}/{x}/{y}.png', {
```
@chegejames Same question here.

ricki-z replied:
Same answer here ;-)

robertprice committed:
Added Eastbourne (UK) to places.js

ricki-z committed:
Merge pull request #60 from robertprice/master

Added Eastbourne (UK) to places.js

ricki-z committed:
new places

ricki-z committed:
added new places

Commit:
all missing zapp

ricki-z committed:
graphs changed from image to iframe

Commit:
Merge branch 'master' of github.com:opendata-stuttgart/feinstaub-map

ricki-z committed:
new countries

orangesonappletrees committed:
Dynamic hex radius for different zoom levels

Dynamic hex radius for different zoom levels to give a less cluttered overview of the sensors as well as making sure that the hex grid does not cover the whole world when zoomed out.

orangesonappletrees committed:
Added Sligo, Ireland

Added Sligo, Ireland, to places.

orangesonappletrees committed:
Added comma after Sligo

ricki-z committed:
new places

ricki-z committed:
australian cities added

ricki-z committed:
Merge pull request #64 from orangesonappletrees/master

Dynamic hex radius for less cluttered map

DavidLemayian commented:
Done by https://github.com/CodeForAfricaLabs/sensors.AFRICA-AQ-map/commit/276e9cf5bc332e0df79f58655f8cb370bde7809b
